MAUI/Picker/dealing-with-header-and-footer.md

```diff
@@ -0,0 +1,314 @@
+
+SfPicker allows customizing background, text style.
+
+### Background
+
+Header’s background color can be customized by setting `SfPicker.HeaderView.Background` property.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+              xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+              xmlns:sfPicker="clr-namespace:Syncfusion.Maui.Picker;assembly=Syncfusion.Maui.Picker"
+              x:Class="Picker_29.MainPage">
+
+      <sfPicker:SfPicker x:Name="picker">
+          <sfPicker:SfPicker.HeaderView>
+              <sfPicker:PickerHeaderView Background="Gray"/>
+          </sfPicker:SfPicker.HeaderView>
+      </sfPicker:SfPicker>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+    picker.HeaderView.Background=Brush.Gray;
+
+{% endhighlight %}
+{% endtabs %}
+
+### Header text style
+
+The .NET MAUI Picker control, Header text style such as text color, font size, font family, font attributes can be customize as shown in the following code.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+                xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+                xmlns:sfPicker="clr-namespace:Syncfusion.Maui.Picker;assembly=Syncfusion.Maui.Picker"
+                xmlns:local ="clr-namespace:Picker_29"
+                x:Class="Picker_29.MainPage">
+
+        <sfPicker:SfPicker>
+            <sfPicker:SfPicker.HeaderView>
+                <sfPicker:PickerHeaderView>
+                    <sfPicker:PickerHeaderView.TextStyle>
+                        <sfPicker:PickerTextStyle TextColor="Gray" 
+                        FontSize="18" 
+                        FontAttributes="Italic"/>
+                    </sfPicker:PickerHeaderView.TextStyle>
+                </sfPicker:PickerHeaderView>
+            </sfPicker:SfPicker.HeaderView>
+        </sfPicker:SfPicker>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+    using Syncfusion.Maui.Picker;
+    using System.Collections.ObjectModel;
+    namespace Picker_29;
+    
+    public partial class MainPage : ContentPage
+    {
+        public MainPage()
+        {
+            InitializeComponent();
+            SfPicker picker = new SfPicker();
+            picker.HeaderView.TextStyle = new PickerTextStyle()
+            {
+                TextColor = Colors.Gray,
+                FontSize = 18,
+                FontAttributes = FontAttributes.Italic
+            };
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Enable or disable footer
+
+SfPicker allows enabling or disabling the footer section by setting  `SfPicker.FooterView.Height` property grater than 0. Default value of `SfPicker.FooterView.Height` property is 0.
```

Comment:
SfPicker allows enabling or disabling the footer section by setting the `SfPicker.FooterView.Height` property greater than 0. The default value of the `SfPicker.FooterView.Height` property is 0.

Reply:
Content modified